docs/5.0/preview/database-access.md

```diff
@@ -38,6 +38,144 @@ With Database Access users can:
+
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Effect": "Allow",
+         "Action": [
+             "rds-db:connect"
+         ],
+         "Resource": [
+             "arn:aws:rds-db:<region>:<account-id>:dbuser:<resource-id>/*"
+         ]
+      }
+   ]
+}
+```
+
+!!! note "Resource ID"
+    Database resource ID is shown on the Configuration tab of a particular
+    database instance in RDS control panel, under "Resource id". For regular
+    RDS database it starts with `db-` prefix. For Aurora use database cluster
+    resource ID (`cluster-`), not individual instance ID.
+
+Finally, connect to the database and create a database account with IAM auth
+support (or update an existing one). Once connected, execute the following
+SQL statements to create a new database account and allow IAM auth for it:
+
+```sql
+CREATE USER alice;
+GRANT rds_iam TO alice;
+```
+
+For more information about connecting to the PostgreSQL instance directly,
+see Amazon [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html).
+
+See a more detailed description of these steps in the [reference](#aws-rdsaurora-postgresql) below.
+
+### Step 2/3. Setup Teleport
+
+Download the latest version of Teleport for your platform from [downloads page](https://goteleport.com/teleport/download).
+Database Access is available starting from `6.0.0-alpha.1` release.
+
+Start Teleport using the following command and point it to your Aurora database
+instance. Make sure to update the database endpoint and region appropriately.
+
+```shell
+sudo teleport start \
+  --roles=proxy,auth,database \
+  --db-name=aurora \
+  --db-protocol=postgres \
+  --db-uri=postgres-aurora-instance-1.abcdefghijklm.us-west-1.rds.amazonaws.com:5432 \
+  --db-aws-region=us-west-1
+```
+
+!!! note "AWS credentials"
+    The node where the Teleport process is started should have AWS credentials
+    configured with the policy from [step 1](#step-13-setup-aurora).
+
+Create a Teleport user that is allowed to connect to a particular database
+(e.g. `postgres`) within the Aurora instance as a particular database account
+(e.g. `alice`).
+
+```shell
+sudo tctl users add alice root \
+  --db-names=postgres \
+  --db-users=alice
+```
+
+### Step 3/3. Connect to Database
+
+Now that Aurora is configured with IAM authentication, Teleport is running and
+the local user is created, we're ready to connect to the database.
+
+Log into Teleport with the user we've just created.
+
+!!! note "Teleport certificate"
+    For simplicity, we're using an `--insecure` flag to accept Teleport's
+    self-signed certificate. For production usage make sure to configure proxy
+    with a proper certificate/key pair. See Teleport's general
+    [quickstart guide](../quickstart.md#step-1c-configure-domain-name-and-obtain-tls-certificates-using-lets-encrypt).
+
+```shell
+tsh login --insecure --proxy=localhost:3080 --user=alice
```

Comment:
We should add a note that during beta customers will need to obtain `tsh` version 6.0

Reply:
Right, added above.